tw-tasks-core/src/main/java/com/transferwise/tasks/triggering/KafkaTasksExecutionTriggerer.java

```diff
@@ -180,7 +181,10 @@ public void trigger(BaseTask task) {
     }
 
     if (BooleanUtils.isTrue(bucketsManager.getBucketProperties(processingBucketId).getTriggerInSameProcess())) {
-      TaskTriggering taskTriggering = new TaskTriggering().setTask(task).setBucketId(processingBucketId);
+      TaskTriggering taskTriggering = new TaskTriggering()
+          .setTask(task)
+          .setBucketId(processingBucketId)
+          .setTriggerAt(Instant.now());
```

Comment:
If task triggered in same process - ran on the pod that triggered it without kafka just use time now